src/app/App.tsx

```diff
@@ -76,24 +62,6 @@ export const App = (): JSX.Element => {
     }
   }, [dispatch, connected]);
 
-  // Redirect to the intro pages if not completed.
-  useEffect(() => {
```

Comment:
This has been moved to the `Header` component. It could've stayed here but I think it makes more sense for routing stuff to live in `Header`